src/pages/Lend/LendPage.vue

```diff
@@ -6,16 +6,19 @@
 				<p class="small-12 xlarge-9">Each Kiva loan helps people build a better
 				future for themselves and their families.</p>
 			</div>
+			<BorrowerInfo />
 		</div>
 	</www-page>
 </template>
 
 <script>
 import WwwPage from '@/components/WwwFrame/WwwPage';
+import BorrowerInfo from '@/components/LoanCards/BorrowerInfo';
```

Comment:
@BoulderBrains -Do you think we ought to build at least a first cut loan-card component (just a div with a border around it), and have that be the component that specifies and holds all the loan card sub-components? So LendPage.vue would only import LoanCard.

Reply:
Oh - I see you already did gridLoanCard

Reply:
so maybe revise this with that context

Reply:
I believe my second commit has this covered. I've replaced BorrowerInfo with GridLoanCard, then piped the BorrowerInfo through GridLoanCard.